hota/Mods/Artifacts/Content/config/hota/artifacts/capeOfSilence.json

```diff
@@ -1,22 +1,26 @@
 {
 	"capeOfSilence" : {
+		"type" : [ "HERO" ],
+		"slot" : "SHOULDERS",
+		"class" : "MAJOR",
+		"value" : 7500,
 		"bonuses" : [
-			{ 
-				"propagator": "BATTLE_WIDE",
-				"propagationUpdater" : "BONUS_OWNER_UPDATER",
+			{
 				"type" : "LEVEL_SPELL_IMMUNITY", 
-				"val" : 1 
+				"val" : 2,
```

Comment:
Not sure how this bonus works, but it does block *both* level 1 and level 2 spells now. Correct?

Reply:
I looked for red dragons as a model and there it was:
```
			"spellImmunity" :
			{
				"type" : "LEVEL_SPELL_IMMUNITY",
				"val" : 3
			}
```
should work the same. will recheck all once artifacts work again.